docs/rules/no-unregistered-components.md

```diff
@@ -0,0 +1,132 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/no-unregistered-components
+description: disallow using components that are not registered inside templates
+---
+# vue/no-unregistered-components
+> disallow using components that are not registered inside templates
+
+## :book: Rule Details
+
+This rule reports components that haven't been registered and are being used in the template.
```

Comment:
Could you document that this rule cannot check globally registered components and components registered in mixins?

Reply:
I've added a warning box telling this and explaining that `component`, `suspense` and `teleport` are exceptions for this.